paddle/fluid/operators/detail/grpc_service.h

```diff
@@ -62,6 +62,7 @@ class SerializationTraits<paddle::operators::detail::VariableResponse> {
       paddle::operators::detail::GrpcByteSource source(buffer);
       int ret = msg->Parse(&source);
       if (ret != 0) {
+        PADDLE_ENFORCE(false, "variable parse error!");
```

Comment:
It seems ```result``` is expected to be returned and the program is expected to fail gracefully? So I think we don't want to add a ENFORCE here?

Reply:
Thanks.